tests/test_wave_01.py

```diff
@@ -118,12 +117,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) is 0
     assert len(updated_data["watched"]) is 1
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
In this case where the code is checking individual keys of a dictionary, I recommend including all of the relevant keys.  
```suggestion
    watched_movie = updated_data["watched"][0]
    assert watched_movie["title] == MOVIE_TITLE_1
    assert watched_movie["rating"] == RATING_1
    assert watched_movie["genre"] == GENRE_1
```